spaces/S000113/properties/P000116.md

```diff
@@ -0,0 +1,15 @@
+---
+space: S000113
+property: P000116
+value: true
+refs:
+- doi: 10.1007/978-1-4612-4190-4
+  name: Classical Descriptive Set Theory (Kechris)
+---
+
+By Theorem 3.11 of {{doi:10.1007/978-1-4612-4190-4}}, and because {S176|P116}, it suffices
+to show {S113} is $G_\delta$ in {S176}. The closure of the space in the plane
```

Comment:
A $G_\delta$ subspace of a Polish space is Polish, so I think we just need to note {S113} is $G_\delta$ in {S114} by your argument, and {S114|P116}.